src/main/java/org/synyx/urlaubsverwaltung/web/MenuDataProvider.java

```diff
@@ -48,6 +53,11 @@ public void postHandle(@NonNull HttpServletRequest request, @NonNull HttpServlet
             modelAndView.addObject("menuHelpUrl", menuProperties.getHelp().getUrl());
             modelAndView.addObject("navigationRequestPopupEnabled", popupMenuEnabled(user));
             modelAndView.addObject("navigationOvertimeItemEnabled", overtimeEnabled(user));
+
+            modelAndView.addObject("isBoss", user.hasRole(BOSS));
+            modelAndView.addObject("isOffice", user.hasRole(OFFICE));
+            modelAndView.addObject("isDepartmentHead", user.hasRole(DEPARTMENT_HEAD));
+            modelAndView.addObject("isSecondStageAuthority", user.hasRole(SECOND_STAGE_AUTHORITY));
```

Comment:
Das würde ich nicht so weiterführen wollen mit diesem `isFoo`. Wenn wir hier schon coden jetzt, dann würde ich direkt im jeweiligen ViewController schauen welches Feature / welche Daten etc in der View angezeigt werden dürfen. Nicht die View (lies: das HTML Template) soll das entscheiden, sondern der Controller.